packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -124,7 +133,9 @@ describe('NotificationDetail Page', () => {
   });
 
   test('renders NotificationDetail page without payment box if creditorTaxId is empty', async () => {
-    result = await renderComponent(overrideNotificationMock({recipients: [{payment: { creditorTaxId: '' }}]}));
```

Comment:
as before, change the test. the previous test can be payments null and this one can be payments empty